openwisp_controller/config/settings.py

```diff
@@ -77,3 +77,8 @@ def get_setting(option, default):
         "WHOIS_GEOIP_ACCOUNT and WHOIS_GEOIP_KEY must be set "
         + "when WHOIS_ENABLED is True."
     )
+APPROXIMATE_LOCATION_ENABLED = get_setting("APPROXIMATE_LOCATION_ENABLED", False)
+if APPROXIMATE_LOCATION_ENABLED and not WHOIS_ENABLED:
+    raise ImproperlyConfigured(
+        "WHOIS must be enabled before enabling APPROXIMATE_LOCATION globally"
+    )
```

Comment:
question:- Do we need to have these tests in both places, i.e. multitenacy and settings?